modules/mocked_authentication/lib/mocked_authentication.rb

```diff
@@ -3,4 +3,8 @@
 require 'mocked_authentication/version'
 require 'mocked_authentication/engine'
 
-module MockedAuthentication; end
+module MockedAuthentication
+  def self.mockable_env?
+    %w[test localhost development staging].include?(Settings.vsp_environment)
```

Comment:
I was going to say we shouldn't have staging included here, but it looks like we specifically added staging to the envs that we attach the mocked auth route to in a PR about a year ago. Do you remember why?

Reply:
staging was added for the `vets-api-mockdata` github action that updates user info from staging MPI. The routes are further restricted in the mocked_authentication [routes](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/mocked_authentication/config/routes.rb#L8)